source/utilities.tex

```diff
@@ -11036,7 +11037,8 @@
 Attempts to allocate a single block larger than this threshold
 will be allocated directly from the upstream memory resource.
 If \tcode{largest_required_pool_block} is zero or
-is greater than an implementation-defined limit, that limit is used instead.
+is greater than an \impldef{largest allocation satisfied directly by a pool}
```

Comment:
I don't think this is quite right; the thing that's implementation-defined is the largest value that is supported as the largest allocation to be satisfied by a pool. But some less-confusing way to word that would be preferred :)